resources/lang/en/activity.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+return [
+    'login'                         => 'Logged In',
+    'registration'                  => 'Registration',
+    'view_home'                     => 'Viewing Home',
+    'view_searches'                 => 'Viewing Searches',
+    'view_lost_people_detail'       => 'Viewing Lost People Details',
+    'view_lost_people_detail'       => 'Viewing User Details',
```

Comment:
'view_lost_people_detail' duplicate.

I think it should probably be 'view_user_details'

Reply:
Yes, you are correct. I made the same mistake here.